packages/mermaid/src/diagrams/common/commonTypes.ts

```diff
@@ -56,3 +58,13 @@ export type D3ImageElement = d3.Selection<SVGImageElement, unknown, Element | nu
 export type D3TextElement = d3.Selection<SVGTextElement, unknown, Element | null, unknown>;
 
 export type D3TSpanElement = d3.Selection<SVGTSpanElement, unknown, Element | null, unknown>;
+
+export interface CommonDb extends DiagramDB {
```

Comment:
```suggestion
export interface CommonDB extends DiagramDB {
```

Reply:
Since we will remove the `db`, should we remove this also?